### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by jphalip**

### CODE REVIEW

File: README-template.md [hunk 1]
```diff
@@ -426,11 +435,165 @@ Please note there are a few caveats:
...
+
+Sample Hive query to read data:
+
+```sql
+SELECT *
+FROM mydb.my_parquet_table
+WHERE city = "Paris"
+```
+
+For more information, refer to the official BigQuery documentation on
```
Comment (anoopj) [inline README-template.md:589]: Also please add a reference to https://cloud.google.com/bigquery/docs/create-cloud-storage-table-biglake

File: README-template.md [hunk 2]
```diff
@@ -426,11 +435,165 @@ Please note there are a few caveats:
...
+```
+
+## Read consistency
+
+As stated in the [BigQuery Storage API documentation](https://cloud.google.com/bigquery/docs/reference/storage),
+read sessions are based on a snapshot isolation model (Note: this is unrelated to
+[BigQuery table snapshots](https://cloud.google.com/bigquery/docs/table-snapshots-intro)). All
+consumers read based on a specific point in time. The default snapshot time is based on the read
+session creation time (i.e. when the `SELECT` query is initiated), but consumers may read data from an
+earlier snapshot.
```
Comment (anoopj) [inline README-template.md:464]: Do we allow this through Hive connector? If so, please add an example.

**Review by anoopj — APPROVED** [review]
LGTM

**Review by yigress — APPROVED** [review]

File: README-template.md (hunk 2 above)
Comment (jphalip) [reply README-template.md]: Ok, so time travel isn't supported by the connector yet. I've clarified the documentation.
The feature should be easy to add some customers explicitly ask for it.

**Pull request merged**